user-interface/src/data-verification/TransferOrderAccordion.tsx

```diff
@@ -229,18 +206,11 @@ export function TransferOrderAccordion(props: TransferOrderAccordionProps) {
     setOrderTransfer(updated);
   }
 
-  function handleSuggestedCaseSelection(bCase: CaseDetailType) {
+  function handleSuggestedCaseSelection(bCase: CaseSummary) {
     if (bCase) {
       const updated = { ...orderTransfer };
       // Remove the division prefix to be consistent with case entry view.
-      updated.newCase = {
-        caseId: getCaseNumber(bCase.caseId),
-        courtDivision: bCase.courtDivision,
-        courtDivisionName: bCase.courtDivisionName,
-        courtName: bCase.courtName,
-        regionId: bCase.regionId,
-        regionName: bCase.regionName,
-      };
+      updated.newCase = bCase;
```

Comment:
I'm not sure why the comment is here, but the code in main does indeed remove the division code from the caseId, but what are we using this for? Why would we want to create a case object with just a partial caseId? We no longer are, so if we needed to in the first place, we should revert this change. That said, I think for most uses, we should only be stripping the code out before displaying the case number.